taqueria-plugin-ligo/compile.ts

```diff
@@ -158,25 +159,27 @@ const compileContractWithStorageAndParameter = async (parsedArgs: Opts, sourceFi
 	const contractCompileResult = await compileContract(parsedArgs, sourceFile);
 	if (contractCompileResult.artifact === COMPILE_ERR_MSG) return [contractCompileResult];
 
-	const storagesFile = `${removeExt(sourceFile)}.storages${extractExt(sourceFile)}`;
-	const parametersFile = `${removeExt(sourceFile)}.parameters${extractExt(sourceFile)}`;
-	const storagesFilename = getInputFilename(parsedArgs, storagesFile);
-	const parametersFilename = getInputFilename(parsedArgs, parametersFile);
+	const storageListFile = `${removeExt(sourceFile)}.storageList${extractExt(sourceFile)}`;
+	const parameterListFile = `${removeExt(sourceFile)}.parameterList${extractExt(sourceFile)}`;
+	const storageListFilename = getInputFilename(parsedArgs, storageListFile);
+	const parameterListFilename = getInputFilename(parsedArgs, parameterListFile);
 
-	const storageCompileResult = await access(storagesFilename)
-		.then(() => compileExprs(parsedArgs, storagesFile, 'storage'))
+	const storageCompileResult = await access(storageListFilename)
+		.then(() => compileExprs(parsedArgs, storageListFile, 'storage'))
 		.catch(() => {
-			// sendWarn(
-			// 	`Note: storage file associated with "${sourceFile}" can't be found. You should create a file "${storagesFile}" and define initial storage values as a list of LIGO variable definitions. e.g. "let STORAGE_NAME: storage = LIGO_EXPR" for CameLigo`,
-			// )
+			sendWarn(
+				`Note: storage file associated with "${sourceFile}" can't be found, so we just created "${storageListFile}" for you. Use this file to define initial storage values as a list of LIGO variable definitions, the first of which will be considered the default storage. e.g. "let STORAGE_NAME: STORAGE_TYPE = LIGO_EXPR" for CameLigo syntax\n`,
```

Comment:
For second person narration, we need to avoid the use of "we, I":

`Note: storage file associated with "${sourceFile}" can't be found, so "${storageListFile}" has been created for you. Use this file to define initial storage values as a list of LIGO variable definitions, the first of which will be considered the default storage. e.g. "let STORAGE_NAME: STORAGE_TYPE = LIGO_EXPR" for CameLigo syntax\n`,

Reply:
done